examples/nextjs/api/rust.rs

```diff
@@ -1,16 +1,13 @@
 use serde_json::json;
 use std::time::Instant;
-use vercel_runtime::{
-    lambda_http::{http::StatusCode, Error as LambdaError, Response},
-    run, Error, IntoResponse, Request,
-};
+use vercel_runtime::{run, Body, Error, Request, Response, StatusCode};
 
 #[tokio::main]
-async fn main() -> Result<(), LambdaError> {
+async fn main() -> Result<(), Error> {
     run(handler).await
 }
 
-pub async fn handler(_req: Request) -> Result<impl IntoResponse, Error> {
```

Comment:
We should now specify this in the response builder as this was previously handled by `into_response`.

```
header(http::header::CONTENT_TYPE, "application/json")
```